app/models/activity.rb

```diff
@@ -1,8 +1,10 @@
 class Activity < PaperTrail::Version
-  default_scope { where(item_type: "Result") }
+  default_scope { where(item_type: ["Result", "Assignment", "OutcomeCoverage"]) }
 
   belongs_to :user, foreign_key: :whodunnit
   belongs_to :result, foreign_key: :item_id
+  belongs_to :assignment, foreign_key: :item_id
+  belongs_to :outcome_coverage, foreign_key: :item_id
```

Comment:
These end up being a lie in some cases, right? For instance, if I get an instance of an activity for an Assignment, I can call `activity.result` and either get nil (if there's no result with that same id) or a result that just so happens to have the same id as an assignment.

The `belongs_to :result` worked before because we constrained it so they were all result activities. I think if we expand this model to cover more types, we have to embrace the existing polymorphic `item` association to get at the thing the activity points to.